test/consolInterfaceDev.jl

```diff
@@ -4,7 +4,7 @@ using Neo4j
 using DistributedFactorGraphs
 using Pkg
 using Dates
-
+using UUIDs
```

Comment:
not sure if we need to update Project.toml extras for UUID test dependency?

Reply:
Should already be there?

Reply:
We didn't include Dates with no problems, so I would say no.